### ISSUE TITLE
Move "Insert snippet" to "Add media" row

### ISSUE THREAD

**Issue by davdebcom** [davdebcom/post-snippets#91]

**Comment by heyjoecampbell** [comment]
I agree, as it would enable the button to be visible for both WYSIWG and Text Editor view.

![image](https://user-images.githubusercontent.com/8339561/44662346-8b9c2c00-a9db-11e8-882c-1934f328dc00.png)
